lib/src/RTCSession.dart

```diff
@@ -2266,8 +2266,8 @@ class RTCSession extends EventEmitter {
 
       // Be ready for 200 with SDP after a 180/183 with SDP.
       // We created a SDP 'answer' for it, so check the current signaling state.
-      //if (this._connection.signalingState == RTCSignalingState.RTCSignalingStateStable)
-      /*{
+      if (this._connection.signalingState ==
+          RTCSignalingState.RTCSignalingStateStable) {
```

Comment:
It works if this line is changed to 

> RTCSignalingState.RTCSignalingStateHaveLocalOffer)  {

Reply:
It looks wrong. If the status is not RTCSignalingStateHaveLocalOffer after receiving 180/183 with SDP, maybe we should check if the early media is properly connected.